### DISCUSSION TITLE
Flatpak distribution?

### DISCUSSION THREAD

**Discussion by ivan-gj** [jorio/gitfourchette#3] [Ideas]
I think this project could benefit a lot from the flatpak distribution model, so users can get timely updates, or hold onto a specific version.

**Comment by jorio** [comment]
Absolutely! I've actually prepared a working flatpak already and I'm planning to submit it soon. Will keep you posted here.

**Comment by jorio** [comment]
[It's ready!](https://flathub.org/apps/org.gitfourchette.gitfourchette)

```
flatpak install flathub org.gitfourchette.gitfourchette
```
